master/docs/manual/cfg-global.rst

```diff
@@ -276,27 +276,39 @@ Data Lifetime
 .. bb:cfg:: changeHorizon
 .. bb:cfg:: buildHorizon
 .. bb:cfg:: logHorizon
+.. bb:cfg:: JanitorConfigurator
 
 Horizons
 ++++++++
 
 ::
 
-    c['changeHorizon'] = 200
-    c['buildHorizon'] = 100
-    c['logHorizon'] = 40
-    c['buildCacheSize'] = 15
+    from buildbot.plugins import util
+    from datetime import timedelta
+
+    # at the end of your master.cfg
+    # configure a janitor which will delete all logs older than one month, and will run on sundays at noon
+    util.JanitorConfigurator(c,
```

Comment:
Indeed, the goal is to extend it with more features.

At the moment I guess we need the bare minimum to not explode your hdd, and we will see if people ask for more.